concepts.md

```diff
@@ -207,6 +207,16 @@ Each half contains the name of the authentication method, a value (in case of em
 
 A [Public Provisional Identity] can be generated from a [Secret Provisional Identity] and consists of the [Secret Provisional Identity] without its secrets parts.
 
+### TLS
+
+Transport Layer Security is always used between *Tanker Core* and *Tanker Server* to exchange information.
+This guarantee secrecy of the data exchanged.
+
+### Resource ID
+
+A randomly generated unique ID part of an encrypted data.
```

Comment:
it is not always random

Reply:
You talked about stream i supposed. But stream ID are not resource ID, right ?